### DISCUSSION TITLE
Preselect Refinement / Filter Option in V4

### DISCUSSION THREAD

**Discussion by adamsdatconed** [microsoft-search/pnp-modern-search#1454] [Q&A] [answered]
Is there a way to pass a query string to the v4 filters like we could in v3 to preselect a refinement option?  In v3 you could pass a querystring e.g. ?filters=[{"n":"FileType","t":["ppt*"]}] and preselect the refinement option for you, but I haven't found a way to do it in v4.  Is this possible?  Thank you!

**Comment by FranckyC** [comment, marked as answer]
Hi @adamsdatconed , something like this https://github.com/microsoft-search/pnp-modern-search/discussions/1367? Will be merged in the upcoming release soon.

**Reply by adamsdatconed** [reply]
Great thank you for the quick response.  Awesome work on this repo!

**Reply by mionline** [reply]
<3
